core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.module.ts

```diff
@@ -41,12 +45,16 @@ import { platformFeatureInitFactory, PlatformFeatureService } from
   declarations: [
     OppiaAngularRootComponent,
     CkEditorCopyToolbarComponent,
-    TranslationLanguageSelectorComponent
+    TranslationLanguageSelectorComponent,
+    LoginRequiredMessageComponent,
+    LoginRequiredModalContent
   ],
   entryComponents: [
     OppiaAngularRootComponent,
     CkEditorCopyToolbarComponent,
-    TranslationLanguageSelectorComponent
+    TranslationLanguageSelectorComponent,
+    LoginRequiredMessageComponent,
+    LoginRequiredModalContent
```

Comment:
Apart from OppiaAngularRootComponent, keep the rest lexicographically sorted.

Reply:
Got it, they're now sorted.